backend/src/Machine/machine.service.ts

```diff
@@ -0,0 +1,55 @@
+import MachineModel from './machine.models';
+
+class Service {
+    public async getAllMachines(): Promise<MachineModel[]> {
+        const allMachines = await MachineModel.getAll();
+        return allMachines;
+    }
+
+    public async createNewMachine(status: string, numberOrderCompleted: number): Promise<number> {
```

Comment:
i think create machine should not take any param and default to 'free' and 0

Reply:
Done!